src/assets.ts

```diff
@@ -146,7 +147,7 @@ function loadAndMergeMultipleImgs(srcs: string[]): Promise<ImageData> {
 						ctx.drawImage(img, 0, 0);
 					}
                 });
-                resolve(ctx.getImageData(0, 0, images[0].width, images[0].height));
```

Comment:
oh sorry i missed this before, I think `ImageData` should be a better return type than `HTMLCanvasElement`